lib/chef-dk/skeletons/code_generator/recipes/cookbook.rb

```diff
@@ -130,6 +130,6 @@
 
 if context.enable_delivery
 
-  include_recipe "code_generator::build_cookbook"
+  include_recipe "#{cookbook_name}::build_cookbook"
```

Comment:
you can just use `include_recipe "::build_cookbook"`

i meant that if `code_generator` appeared anywhere else that the cookbook_name var could be used to replace it.